eng/dependabot/independent/dependabot.csproj

```diff
@@ -0,0 +1,6 @@
+<!-- This isn't a real project, but Dependabot requires a project. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Do we need a target framework here?

Reply:
Unfortunately yes. dependabot now injects a custom build target into projects it tries to update and effectively uses msbuild on it to build the custom target, so we need to specify a tfm.